software/base/src/test/java/brooklyn/entity/software/MachineLifecycleEffectorTasksTest.java

```diff
@@ -23,11 +23,17 @@
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
+import brooklyn.entity.basic.BasicEntityImpl;
+import brooklyn.entity.basic.Lifecycle;
+import brooklyn.entity.basic.SoftwareProcess;
 import brooklyn.entity.basic.SoftwareProcess.StopSoftwareParameters.StopMode;
 
 public class MachineLifecycleEffectorTasksTest {
     public static boolean canStop(StopMode stopMode, boolean isEntityStopped) {
-        return MachineLifecycleEffectorTasks.canStop(stopMode, isEntityStopped);
+        BasicEntityImpl entity = new BasicEntityImpl();
```

Comment:
ok for now, but at some point calling the entity constructor directly may give us very surprising subsequent behaviour (e.g. because not managed, and no management context injected).